sphinx/texinputs/latexmkrc_t

```diff
@@ -11,7 +11,7 @@ $pdflatex = 'xelatex ' . $ENV{'LATEXOPTS'} . ' %O %S';
 $lualatex = 'lualatex ' . $ENV{'LATEXOPTS'} . ' %O %S';
 $xelatex = 'xelatex --no-pdf ' . $ENV{'LATEXOPTS'} . ' %O %S';
 {% if xindy_use -%}
-$makeindex = 'xindy ' . $ENV{'XINDYOPTS'} . ' %O -o %D %S';
+$makeindex = '[ ! -s %S ] && : > %D || xindy ' . $ENV{'XINDYOPTS'} . ' %O -o %D %S';
```

Comment:
How about this instead?
```
$makeindex = 'internal xindy ' . $ENV{'XINDYOPTS'} . ' %O -o %D %S';
sub xindy {
  my @args = @_;
  if (-s $args[-1]) {
    # create an empty .ind file if .idx is empty
    open(FH, ">" . $args[-2]);
    close(FH);
  } else {
    return system("xindy", @args);
  }
}
```

Unfortunately, I don't have enough time until this weekend. So this code is not tested (sorry!).
But I think it is better to implement this with perl script instead of shell script to support windows platform (in future).

Reply:
+1 for this idea! currently it does not work in my testing (case of empty file not detected) but I will work a bit on it. Unfortunately I have little time until September but I will give it a try now.

Reply:
@tk0miya I obtain success with 

```
$makeindex = 'internal xindy ' . $ENV{'XINDYOPTS'} . ' %O -o %D %S';
sub xindy {
  my @args = @_;
  if (-z $args[-1]) {
    # create an empty .ind file if .idx is empty
    print "Raw index file is empty, ignoring.\n";
    return system(": > $args[-2]");
  }
  else
  {
    return system("xindy", @args);
  }
}
```

but I needed some system call because Latexmk analyses exit status (there is a division at line 7020 in latexmk version ``25 May 2018. Version 4.56``). And I am no-ob in Perl. Also I changed to `-z` to `-s`. Surely there is better way.